pages/satistics.js

```diff
@@ -166,6 +171,64 @@ export default function Satistics ({ ssrData }) {
 
   const { walletHistory: { facts, cursor } } = data || ssrData
 
+  const me = useMe()
+  const getCsvBtnClass = () => {
+    return me.csvRequest === CsvRequest.NO_REQUEST
+      ? 'btn-grey-darkmode'
+      : me.csvRequestStatus === CsvRequestStatus.FULL_REPORT
+        ? 'btn-success'
+        : 'btn-danger'
+  }
+  const getIconClass = () => {
```

Comment:
Why are this and `getCsvBtnClass` functions?

Reply:
At some point refreshes weren't happening when I needed them, so I put functions there (but I am not even sure anymore whether making them functions did anything for my original problem). Anyway, it is working without them, so that's good.